src/components/AnalysisPage/GeoAnalysisBox/GeoAnalysisBox.jsx

```diff
@@ -21,6 +21,15 @@ const GeoAnalysisBox = ({ partId }) => {
     setNumShells(null);
   }, [partId]);
 
+  useEffect(() => {
+    if (autoPopAnalysis) {
+      setVolume(autoPopAnalysis.volume);
+      setSurfaceArea(autoPopAnalysis.surface_area);
+      setNumBoundaryEdges(autoPopAnalysis.num_boundary_edges);
+      setNumShells(autoPopAnalysis.num_shells);
+    }
+  }, [autoPopAnalysis]);
+
   useEffect(() => {
     console.log(geoData);
     if (geoData) {
```

Comment:
@rknizzle 

I was planning on using `autoPopAnalysis` for both the geo-box and thickness-box. My thinking was that it would be completely separate from geoData and what will be thinknessData. 

Where geo-data is the information gained from the initial running of the analysis, so It would only need to be used and accessible to geoAnalysisBox and its start button.

When the thickness analysis box starts rolling i'll look into this more!

Reply:
Cool ya thats one way to do it. I just think with that method there are basically 2 geoData fields. And whichever one gets used just depends on if the part was just uploaded or if it was loaded from the list of parts. And so you can see like because of having 2 geoData fields you also have to have 2 basically identical useEffects to set the dimensions. So I just think it might be better to just combine them into a single geoData field which would have to live in the analysis page so that both the ViewPartsButton and the GeomBox can set it

Reply:
But I haven't really messed with the code so that could be more complicated than it seems from just reading through. So ya you can just keep both methods in mind as you keep building stuff and see what feels better @ErikAmerico

Reply:
As I'm working through the thickness analysis, I'm seeing where we differed on this. 

My original plan of having autoPopAnalysis take care of both of the analysis boxes isn't going to work as i thought it would. 

So I'm going to get rid of autoPop and then place geoData in the analysisPage like you suggested!

@rknizzle (the sensei)